src/Apps/Legion/Components/WizardShell/WizardShellMockData.ts

```diff
@@ -109,29 +105,24 @@ export const steps: IStepperWizardStep[] = [
     }
 ];
 
-export const cookedData: ICookAssets = {
+export const appData: IAppData = {
```

Comment:
let's reanmae this to `DEFAULT_MOCK_WIZARD_SHELL_DATA`

Reply:
this is internal so I'll remove the export and rename to mockAppData